tests/test_sidecar.py

```
@@ -143,22 +143,22 @@ def test_parse_no_favorite_field(tmp_path: Path) -> None:
-    """Test that 0/0 coordinates are filtered out as unreliable."""
+    """Tester que les coordonnées 0/0 sont filtrées car peu fiables."""
     sample = {
         "title": "geo_zero.jpg",
         "geoData": {"latitude": 0.0, "longitude": 0.0, "altitude": 100.0}
     }
     json_path = tmp_path / "geo_zero.jpg.json"
     json_path.write_text(json.dumps(sample), encoding="utf-8")
     meta = parse_sidecar(json_path)
-    # 0/0 coordinates should be filtered out
+    # Les coordonnées 0/0 doivent être filtrées
     assert meta.latitude is None
```

Ruff (0.12.2)

Use of `assert` detected

(S101)